include/fi.h

```diff
@@ -209,8 +235,12 @@ int fi_rma_target_allowed(uint64_t caps);
         asm(".symver " #name "," #api "@@" DEFAULT_ABI)
 #else
 #  define symver(name, api, ver)
+#ifdef  HAVE_ALIAS_ATTRIBUTE
```

Comment:
I think this could be simplified like:

```
#if defined(HAVE_SYMVER_SUPPORT) && defined(HAVE_ALIAS_ATTRIBUTE)
...define default_symver macro as normal
#else
#define default_symver(name, api)
#endif